pkg/pgmodel/cache/cache.go

```diff
@@ -70,7 +88,7 @@ func (m *MetricNameCache) Set(metric string, tableName string) error {
 	tableBuilder.Grow(len(tableName))
 	tableBuilder.WriteString(tableName)
 	//size includes an 8-byte overhead for each string
-	m.Metrics.Insert(metricBuilder.String(), tableBuilder.String(), uint64(metricBuilder.Len()+tableBuilder.Len()+16))
+	m.Metrics.Insert(metricInfo{metricBuilder.String(), isExemplar}, tableBuilder.String(), uint64(metricBuilder.Len()+tableBuilder.Len()+16))
```

Comment:
Since you are storing `isExemplar` boolean as well, size needs to be increased by 1 byte. Make sure to add that to the comment.